doc/man3/SSL_CONF_cmd.pod

```diff
@@ -85,6 +85,13 @@ its preference list. This usually indicates a client without AES hardware
 acceleration (e.g. mobile) is in use. Equivalent to B<SSL_OP_PRIORITIZE_CHACHA>.
 Only used by servers. Requires B<-serverpref>.
 
+=item B<-allow_ssl_eof>
+
+Some TLS implementation (such as L<openssl-s_server(1)> before the OpenSSL 3.0
+or IIS) do not send close_notify on shutdown.  Such behaviour is treated as
+error since OpenSSL 3.0. Enabling this option allows to treat such behaviour as
```

Comment:
Strictly speaking its still an error even if using this option. It always was. It was just historically reported as SSL_ERROR_SYSCALL with a 0 errno (which is against the documented behaviour for that SSL_ERROR_SYSCALL). What we have changed is that the error is now reported as SSL_ERROR_SSL with a meaningful error on the error stack. So really this option is about getting the legacy error reporting back rather than the new error reporting. The docs should probably explain all of this. I'd also suggest calling the option SSL_OP_LEGACY_SSL_EOF, or similar. It's not really about allowing it - just changing how it is handled.

Reply:
Fixed.